benches/bench_basic.rs

```diff
@@ -22,6 +28,27 @@ fn bench_simple_getsetdel(b: &mut Bencher) {
     });
 }
 
+fn bench_simple_getsetdel_async(b: &mut Bencher) {
+    let client = get_client();
+    let mut core = Core::new().unwrap();
+    let con = client.get_async_connection(&core.handle());
```

Comment:
`get_async_connection` doesn't take a handle anymore, so I guess this all needs to change.

Reply:
Fixed (and I enabled `cargo check` for travis)